### ISSUE TITLE
tensorflow에서 구현된 dropout 분석

### ISSUE THREAD

**Issue by chatterboy** [chatterboy/sad#5]

**Comment by chatterboy** [comment]
tf.layer와 tf.nn에 있을 것 같다. 더 조사해보자.
**참고**
[1] https://github.com/tensorflow/tensorflow/blob/r1.12/tensorflow/python/ops/nn_ops.py#L2259
